action.yml

```
@@ -79,6 +79,75 @@ runs:
         aws_secret_access_key: ${{ inputs.aws_secret_access_key }}
         aws_region: ${{ inputs.aws_region }}
 
+    - name: Render and deploy task definition to Amazon ECS
+      id: deploy-migration-task-definition
```

I thought we agreed on preflight? So please update everything with `migration` in the name to `preflight`.
See: https://github.com/moneymeets/moneymeets-pulumi/pull/660#discussion_r1705132182
```suggestion
      id: deploy-preflight-task-definition
```